src/shared/sam/cli/samCliPackage.ts

```diff
@@ -17,18 +17,21 @@ export class SamCliPackageInvocation {
         private readonly s3Bucket: string,
         private readonly invoker: SamCliProcessInvoker =
         new DefaultSamCliProcessInvoker(),
-        private readonly region: string
+        private readonly region: string,
+        private readonly profile: string
     ) {
     }
 
     public async execute(): Promise<void> {
         const logger: Logger = getLogger()
+
         const { exitCode, error, stderr, stdout }: ChildProcessResult = await this.invoker.invoke(
             'package',
             '--template-file', this.templateFile,
             '--s3-bucket', this.s3Bucket,
             '--output-template-file', this.outputTemplateFile,
-            '--region', this.region
+            '--region', this.region,
+            '--profile', this.profile
```

Comment:
samCliLocalInvoke.test.ts contains unit tests that verify we pass expected values to the invoker.invoke call.
Add a similar one for sam cli package to test profile. Tests can be backfilled for existing untested code later, but incremental changes that can be tested should be tested.

Reply:
✔️